x/valset/client/cli/query_get_latest_published_snapshot.go

```diff
@@ -0,0 +1,39 @@
+package cli
+
+import (
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/palomachain/paloma/x/valset/types"
+	"github.com/spf13/cobra"
+)
+
+func CmdGetLatestPublishedSnapshot() *cobra.Command {
+	cmd := &cobra.Command{
```

Comment:
we dont need the cli command for pigeons to pick up this info, but it won't hurt either :)

Reply:
Super easy to include the CLI command.  I didn't see any harm in it and it makes it easier to debug